src/isar/storage/utilities.py

```diff
@@ -0,0 +1,96 @@
+import json
+from pathlib import Path
+from typing import Any, Tuple
+from uuid import UUID
+
+from isar.models.mission_metadata.mission_metadata import MissionMetadata
+from isar.services.utilities.json_service import EnhancedJSONEncoder
+from robot_interface.models.inspection.inspection import Image, Inspection, ThermalImage
+
+
+def construct_local_paths(
+    inspection: Inspection, metadata: MissionMetadata
+) -> Tuple[Path, Path]:
+    inspection_type: str = get_inspection_type(inspection=inspection)
+
+    folder: Path = Path(str(metadata.mission_id))
+    filename: str = get_filename(
```

Comment:
for consistency should we call it inspection_filename?

Reply:
I don't think so as it is now the filename of both files.

Reply:
Agreed